content/applications/finance/fiscal_localizations/australia.rst

```diff
@@ -2,14 +2,873 @@
+
+.. _australia/buynow_paylater:
+
+Buy Now, Pay Later solutions
+============================
+
+Buy Now, Pay Later solutions are popular payment methods for eShops in Australia. Currently, some of
+these solutions are available via the `Stripe <https://stripe.com/en-au/payments/payment-methods>`_
+and `AsiaPay <https://www.asiapay.com.au/payment.html#option>`_ payment providers.
+
+.. seealso::
+   - `AsiaPay payment acquirer <https://www.odoo.com/documentation/17.0/applications/finance/payment_providers/asiapay.html>`_
+   - `Stripe payment acquirer <https://www.odoo.com/documentation/17.0/applications/finance/payment_providers/stripe.html>`_
+
+POS terminals for Australia
+===========================
+
+Currently, if you wish to have a direct connection between Odoo and your PoS terminal in Australia,
+you *must* have a **Stripe** terminal. Odoo supports the **EFTPOS** payment solution in Australia.
+
+.. note::
+   You do not need a Stripe payment terminal to use Odoo as your main PoS system. The only drawback
+   of not using Stripe is that cashiers will need to enter the final payment amount manually on the
+   terminal.
+
+.. seealso::
+   - `Stripe Odoo Payment Provider <https://www.odoo.com/documentation/17.0/applications/finance/payment_providers/stripe.html>`_
+   - `Stripe Odoo Payment Terminal <https://www.odoo.com/documentation/17.0/applications/sales/point_of_sale/payment_methods/terminals/stripe.html>`_
+   - `Stripe Dashboard <https://dashboard.stripe.com/dashboard>`_
+   - `Stripe Terminal <https://stripe.com/docs/terminal>`_
+
+.. _australia/payroll:
+
+Payroll
+=======
+
+Create your employees
+---------------------
+
+To create an **employee form** and fill in mandatory information, go to the **Employees** app and
+click :guilabel:`Create`. In the :guilabel:`HR Settings` tab, you can input mandatory information:
+
+#. :guilabel:`TFN Status`: if a TFN is provided by the employee, make sure to add it to the “TFN”
+   field
+#. :guilabel:`Non-resident`: if the employee is a foreign resident
+#. :guilabel:`Tax-free Threshold`: if the employee is below or above the threshold
+#. :guilabel:`HELP / STSL`: for all types of study and training support loans (e.g. HELP, VSL, SSL…)
+#. :guilabel:`Medicare levy Deduction`
+#. :guilabel:`Medicare levy Exemption`
+#. :guilabel:`Withholding for Extra Pay`
+#. ...
+
+Manage Australian contracts
+---------------------------
+
+Once the employee form has been created, ensure the contract is enabled  by clicking on the
+:guilabel:`Contracts` smart button, or by going to :menuselection:`Employees --> Employees -->
+Contracts`.
+
+.. note::
+   Only one contract can be active at the same time per employee, but it is possible for an employee
+   to be assigned consecutive contracts during their employment.
+
+Contractual information related to Australia can be found in the following **3** places:
+
+1. Before assigning a salary amount, a pay frequency and other important details to a contract, it
```

Comment:
isolated nubered list (should be #.). it seems like it follows as a bullet list afterwards.